server/statshouse/statshouse-client.cpp

```diff
@@ -7,6 +7,7 @@
 statshouse::TransportUDPBase::MetricBuilder StatsHouseClient::metric(std::string_view name, bool force_tag_host) {
   auto builder = transport.metric(name);
   builder.tag(tag_cluster);
+  builder.tag(tag_dc_name);
```

Comment:
Unnamed tag like this will break all current dashboards, because all other unnamed tags will shift. We need to use named tag here

Reply:
omg, you're right as always. I thought of that and promptly forgot about it. Thanks for noticing that.